jstests/aggregation/isodates.js

```diff
@@ -0,0 +1,243 @@
+/*
```

Comment:
@cswanson310: I've added a new [test file](https://github.com/jigfox/mongo/blob/SERVER-7695-aggregation-operators-for-iso-week/jstests/aggregation/bugs/server7695_isodates.js), which is much more explanatory.